### DISCUSSION TITLE
How to insert images in slides?

### DISCUSSION THREAD

**Discussion by liblaf** [vuepress-theme-hope/vuepress-theme-hope#2203] [Q&A] [answered]
I tried to insert an image with a relative path.

```markdown
![alt](./assets/img/sample.png)
```

However, the resulting image path in HTML remains the same relative path instead of a transformed URL.

How could I insert a local image in slides?

**Comment by Mister-Hope** [comment, marked as answer]
Use absolute links and place them in public folder. OR full links

You should understand that every part between `@slidestart` and `@slideend` is actually passed to and parsed by `reveal.js` instead of being handled by vuepress built-in markdown parser.
